app/config.rb

```diff
@@ -0,0 +1,34 @@
+class Config
+  def initialize(environment)
+    @environment = environment || "development"
+  end
+
+  attr_reader :environment
+
+  def app_root
+    @app_root ||= Pathname.new(File.expand_path(File.join(File.dirname(__FILE__), "..")))
+  end
+
+  def logger
+    @logger ||= begin
+      logfile = File.open(app_root + "log/#{environment}.log", "a")
```

Comment:
Any reason for using `File.join` above but string concatenation here?

Reply:
Good question, it's coming from `email-alert-service` but I think we can make it better, I'll do some refactoring.